spark-bigquery-pushdown/spark-2.4-bigquery-pushdown_2.11/src/main/scala/com/google/cloud/spark/bigquery/pushdowns/Spark24BigQueryPushdown.scala

```diff
@@ -3,12 +3,12 @@ package com.google.cloud.spark.bigquery.pushdowns
 import org.apache.spark.sql.SparkSession
 
 class Spark24BigQueryPushdown extends SparkBigQueryPushdown {
-  override def enable(session: SparkSession): Unit = {
-    // TODO
+  override def enable(session: SparkSession, bigQueryStrategy: BigQueryStrategy): Unit = {
```

Comment:
Why do we need the customer to provide the  strategy? can the enable method create it?

Reply:
The customer does not provide the strategy. Customers only need to call `[BigQueryConnectorUtils.enablePushdownSession](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/pull/547/files#diff-18108bb7ffb8890783416955114b08b6ee27104242b557289d65ed4997921382R10)` and the correct pushdown service will be called based on the Spark and Scala version.